spec/features/stations_spec.rb

```diff
@@ -76,21 +69,24 @@
     expect(page).to have_content 'Station at the End of The World'
   end
 
-  scenario "when I view index I should not see hidden stations" do
-    station = create(:station, show: false)
-    visit stations_path
-    expect(page).to_not have_link station.name
-  end
+  context "hidden stations" do
+    let!(:deactivated_station) { create(:station, status: :deactivated) }
+    let!(:new_station) { create(:station, status: :not_initialized) }
 
-  scenario "when I am signed in as an admin and view index I should see hidden stations" do
-    skip %Q{
-      This works when I manually check in browser but for some reason capybara gets a document without any
-      stations at all. I must be missing something
-    }
-    admin_session
-    #station = create(:station, show: false)
-    #visit stations_path
-    #expect(page).to have_link station.name
+    scenario "when I view index I should not see hidden stations" do
+      station =
+      station2 = create(:station, status: :not_initialized)
```

Comment:
Why is a second not_initialized station created? And what are the purpose of the two variables?